### ISSUE TITLE
Link Account to previous Guest Application(s)

### ISSUE THREAD

**Issue by gwendolyngoetz** [civiform/civiform#8289]
Let says you submit a few applications as a guest. Later you create an account. The previous applications are not bound to your account. We should consider ways to link that previous guest data to the new account.

**Comment by gwendolyngoetz** [comment]
You can currently link applications done as guest to a new account if it's in your active session, but not if you don't have an active session.

**Comment by rockycodes** [comment]
@gwendolyngoetz what do you think about this idea at this point. The only way I could see doing this would be with email or other identifying info, but I feel like we've steered away from that in recent conversations because it would be too easy to accidentally link applications incorrectly.

**Comment by gwendolyngoetz** [comment]
We have no way to guarantee that the original person applying is the person attempting to link an application to an account. **If** we provide a way, it should probably be an admin function to manually link accounts to applications and let the admin determine identity. I just don't see a safe way to do this that doesn't potentially put PII, or more importantly the applicant, at risk.

**Comment by rockycodes** [comment]
I agree with you. The idea of having this be an admin function is a good one. We should explore that when we get to this.
